packages/core/src/routes/session/password.test.ts

```diff
@@ -243,4 +244,48 @@ describe('session -> password routes', () => {
       expect(response.statusCode).toEqual(422);
     });
   });
+
+  describe('POST /session/register/password/check-username', () => {
```

Comment:
/session/register/check-username. no password?

Reply:
Per discussion, won't create a route group for now.